integration_test/third_party_apps_data/applications/iis/metadata.yaml

```diff
@@ -28,3 +28,48 @@ expected_metrics:
   labels:
     http_method: .*
   representative: true
+expected_logs:
+- log_name: iis_access
+  fields:
+  - name: httpRequest.serverIp
+    value_regex: 127.0.0.1:80
+    type: string
+    description: "The server's IP and port that was requested"
+  - name: httpRequest.remoteIp
+    value_regex: 127.0.0.1:80
+    type: string
+    description: "IP of the client that made the requestb v"
+  - name: httpRequest.requestUrl
+    value_regex: /forbidden?something=something
+    type: string
+    description: "Request URL (typically just the path part of the URL)"
+  - name: httpRequest.protocol
+    value_regex: HTTP/1.1
+    type: string
+    description: "Protocol used for the request"
+  - name: httpRequest.requestMethod
+    value_regex: GET
+    type: string
+    description: "HTTP method"
+  # - name: httpRequest.status
+  #   type: number
+  #   description: "HTTP status code"
```

Comment:
Can this be uncommented? IIRC the test framework doesn't support non-strings yet, but we can skip the test by omitting `value_regex` (which is already omitted in the commented portion anyway). This would allow it to still be included in the generated docs later on.

Reply:
Updated